app/core/components/ReviewStars.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import { RatingTotal } from "./RatingTotal"
+import { ReviewCategoryAnswer } from "./ReviewCategoryAnswer"
+
+export const ReviewStars = (props) => {
+  const { reviews, questionCategories } = props
+  const ratingScaleMax = 5
```

Comment:
Note that we are fixing the max rating into 5. Again, we may change this in the future depending on other review questions.